classylss/binding.pyx

```diff
@@ -666,62 +684,66 @@ cdef class Background:
         Density parameter of relativistic (radiation like) component, including
         relativistic part of massive neutrino and massless neutrino.
         """
-        return self.compute_for_z(z, self.ba.index_bg_Omega_r)
+        E = self.efunc(z)
```

Comment:
This is wrong. Omega_r contains the pressure part of massive neutrino which is redshift dependent. So the scaling against (1+z) **4 is wrong.